src/load-config.js

```diff
@@ -53,7 +53,6 @@ if (process.env.ACTUAL_CONFIG_PATH) {
 
 /** @type {Omit<import('./config-types.js').Config, 'mode' | 'dataDir' | 'serverFiles' | 'userFiles'>} */
 let defaultConfig = {
-  loginMethod: 'password',
```

Comment:
Curious why this was removed?

Reply:
it was removed (I don't remember why), but I added it back on the end of the defaultConfig object. I moved it back to the top